src/DataStructures/Tensor/Tensor.hpp

```diff
@@ -260,6 +260,22 @@ class Tensor<X, Symm, IndexList<Indices...>> {
       get(const Tensor<Args...>& t) noexcept;  // NOLINT
   // @}
 
+  // @{
```

Comment:
That is fine. I'm quite certain that this can be dropped from the PR without negative effect now that the `Variables` interface is gone.

Reply:
Yea, I only saw this being used in one place that I noticed :) I understand why you wanted it, but wanted to explain my experience why I think it'll be more trouble than fix problems. (I have a similar complaint about defaulting things because all too often I get that the default thing just doesn't do what I want and it takes me too long to figure out it was because of some hidden option) :)